src/SeleniumLibrary/utils/attrdict.py

```diff
@@ -0,0 +1,16 @@
+class AttrDict(dict):
```

Comment:
Should we rename this in same way as RF DotDic.

Reply:
Although code is simple write few unit test might be good idea.

Reply:
This, I don't agree. I only added AttrDict because of RF 2.8.7 compatibility. It is a temporary workaround.
The original PR had unit tests, but they will not be necessary because it is temporary.

Reply:
There already is some temporary code, waiting for the minimum RF version to raise high enough and they also have unit tests. So having something as temporally and not writing test for it, is not valid reason. Also it would lift the burden from the acceptance test, that one does not have to verify that value is in the right key in the acceptance test level.

Reply:
Couldn't this just wait until we drop RF 2.8.7 support? I don't think this can make it to 3.0 anyway.